proposals/0003_documentation-strategy.md

```diff
@@ -0,0 +1,130 @@
+
+Table 2: The [Brightway](https://github.com/brightway-lca) implementation of the [Diataxis framework](https://diataxis.fr/)
+
+##### Reference
+
+This shall contain the API reference for the various Brightway modules (`bw_agg`, `bw_calc`, `bw_data`, ...). This is compiled automatically from the function docstrings by the [Sphinx package](https://www.sphinx-doc.org/en/master/).
+
+##### Explanation
+
+This shall include the [mathematical formalism of life-cycle assessment](https://doi.org/10.1007/978-94-015-9900-9), some theory of matrix multiplication, etc. The exact contents are to be discussed based on the __"Scope of the Documentation"__ section.
+
+##### Tutorials
+
+This shall include the growing number of high-quality interactive Jupyter Notebooks already available at [`brightway-lca/brightway2/notebooks`](https://github.com/brightway-lca/brightway2/tree/master/notebooks), as well as some of the notebooks [collected as part of the 2022 Brightcon Hackathon](https://github.com/brightway-lca/brightway-training/blob/master/organization/curation/brightway_tutorial_list.xlsx).
+
+##### How-To Guides ("Example Gallery")
+
+This shall contain self-contained examples of Brightway code. Examples shall be collected in the [`example-gallery`](https://github.com/brightway-lca/brightway-examples) repository. They are integrated into the documentation website using a git submodule and the [`myst-nb`](https://myst-nb.readthedocs.io/en/latest/) package.
+
+### Rationale
+
+The `pydata-sphinx-theme` and the `jupyter-book` templates in combination with readthedocs.org are free and open source, future-proof options of hosting documentation.
+
+### Pros and Cons
+
+Pros: An established future-proof standard of documenting code that will result in lower entry barriers for users with little experience in Python.
+
+Cons: Developers will need to write documentation (gasp!).
+
+#### Cons
+
+N/A
+
+### Alternatives
+
+N/A
+
+### Open Issues
+
+Compare the open issues in the respective repositories:
+
+ - [brightway-lca/brightway-documentation](https://github.com/brightway-lca/brightway-documentation)
+ - [brightway-lca/brightway-book](https://github.com/brightway-lca/brightway-book)
+ - [brightway-lca/brightway-live](https://github.com/brightway-lca/brightway-live)
+
+### Non-Goals
+
+N/A
+
+### Test plan and results
+
+Prototypes of [docs.brightway.dev](https://docs.brightway.dev/) and [learn.brightway.dev](https://learn.brightway.dev/) were developed as part of the [Brightcon 2022](http://web.archive.org/web/20220928071701/https://2022.brightcon.link/) Hackathon. The prototype of [live.brightway.dev](https://live.brightway.dev/) was developed as part of the [Brightcon 2023](https://web.archive.org/web/20230918043912/https://2023.brightcon.link/) hackathon.
+
+The readthedocs.org traffic analysis confirms the success of the new documentation (data as per December 2023):
```

Comment:
non-blocking: if we have more up to date figures here, we should add them before merging.